content/patterns/alert/alert-pattern.html

```diff
@@ -14,42 +14,35 @@
   </head>
   <body>
     <main>
-      <h1>Alert Pattern</h1>
+      <h1>アラート パターン</h1>
+      <!-- 翻訳元リビジョン: https://github.com/w3c/aria-practices/tree/d6cf9d8db772603a456e7d7ef52e409d39124301 -->
 
       <section id="about">
-        <h2>About This Pattern</h2>
+        <h2>このパターンについて</h2>
         <p>
-          An <a class="role-reference" href="#alert">alert</a> is an element that displays a brief, important message in a way that attracts the user's attention without interrupting the user's task.
-          Dynamically rendered alerts are automatically announced by most screen readers, and in some operating systems, they may trigger an alert sound.
-          It is important to note that, at this time, screen readers do not inform users of alerts that are present on the page before page load completes.
+          <a class="role-reference" href="#alert">アラート</a>は、利用者のタスクを中断することなく利用者の注意を引く方法によって、簡潔で重要なメッセージを表示する要素です。動的にレンダリングされたアラートは、ほとんどのスクリーンリーダーで自動的にアナウンスされ、一部のオペレーティングシステムでは、アラート音が鳴る場合があります。現時点でスクリーンリーダーは、ページの読み込みが完了する前にページ上に存在するアラートを、利用者に通知しないことに注意することが重要です。
         </p>
         <p>
-          Because alerts are intended to provide important and potentially time-sensitive information without interfering with the user's ability to continue working, it is crucial they do not affect keyboard focus.
-          The <a href="../alertdialog/alertdialog-pattern.html">Alert Dialog Pattern</a> is designed for situations where interrupting work flow is necessary.
+          アラートは、利用者の作業を続行する能力を妨げることなく、重要かつ時間的にセンシティブな恐れのある情報を提供することを目的としているため、キーボードフォーカスに影響を与えないようにすることが非常に重要です。<a href="../alertdialog/alertdialog-pattern.html">アラートダイアログ パターン</a>は、ワークフローを中断する必要がある状況のために設計されたものです。
```

Comment:
「作業を続行する能力を妨げる」はもう少しこなれた日本語にしたい感じがあります。やや意訳になりますが、「作業継続を妨げることなく」くらいで良いのではないでしょうか。

time-sensitive を「時間的にセンシティブな」としていますが、わかりにくいように感じます。ここでは「リアルタイムで、今が旬の情報をそのタイミングで出している」というようなことを言いたいのだと思います。「一刻を争う」という訳し方もあるようなので、ここは「一刻を争うかもしれない」という表現でも良いかもしれません (言い過ぎ?)。

```suggestion
          アラートは、重要かつ一刻を争うかもしれない情報を、利用者の作業継続を妨げることなく提供することを目的としているため、キーボードフォーカスに影響を与えないようにすることが非常に重要です。<a href="../alertdialog/alertdialog-pattern.html">アラートダイアログ パターン</a>は、ワークフローを中断する必要がある状況のために設計されたものです。
```